jax/_src/numpy/ufuncs.py

```diff
@@ -241,13 +241,46 @@ def bitwise_right_shift(x1: ArrayLike, x2: ArrayLike, /) -> Array:
     np.issubdtype(x1.dtype, np.unsignedinteger) else lax.shift_right_arithmetic
   return lax_fn(x1, x2)
 
-@implements(np.absolute, module='numpy')
+
 @partial(jit, inline=True)
 def absolute(x: ArrayLike, /) -> Array:
+  r"""Calculate the absolute value element-wise.
+
+  LAX-backend implementation of :code:`numpy.absolute()`.
+
+  This is the same function as :code:`jnp.abs`.
+
+  Args:
+    x: Input array
+
+  Returns:
+    An array-like object containing the absolute value of each element in :math:`x`,
+    with the same shape as :math:`x`. For complex valued input, :math:`a + ib`,
+    the absolute value is :math:`\sqrt{a^2+b^2}`.
+
+  Example:
+    >>> x1 = jnp.array([5, -2, 0, 12])
+    >>> jnp.absolute(x1)
+    Array([ 5,  2,  0, 12], dtype=int32)
+
+    >>> x2 = jnp.array([[ 8, -3, 1],[ 0, 9, -6]])
+    >>> jnp.absolute(x2)
+    Array([[8, 3, 1],
+           [0, 9, 6]], dtype=int32)
+
+    >>> x3 = jnp.array([3 + 2j, 3 - 4j, -5 + 0j])
+    >>> jnp.absolute(x3)
+    Array([3.6055512, 5.       , 5.       ], dtype=float32)
```

Comment:
Can we stick with integer-valued outputs? The repr of sqrt(13) may differ on different systems, which would cause a doctest failure.